modules/volume_access.cft.yaml

```diff
@@ -394,13 +395,54 @@ Resources:
                     Condition:
                       StringEqualsIgnoreCase:
                         "aws:ResourceTag/CreatedBy": "Sysdig" 
+          AdministrationRole:
```

Comment:
I assume this will create the `AdministrationRole` & `ExecutionRole` in each account of the org?

Reply:
updated